go.mod

```diff
@@ -160,6 +160,7 @@ require (
 
 // TODO: Once the repo is public, remove this so we use a versioned module
 replace (
+	github.com/aws/cluster-api-provider-cloudstack => github.com/wanyufe/cluster-api-provider-cloudstack v0.4.5-rc2.0.20220601153610-4f3e7128e44b
```

Comment:
Can you explain this approach? Is this a temporary measure? I'd rather use the actual repo, and not use your fork in prod. It'll also be confusing when you try to bump the capc version only to realize that nothing is happening because of this replace

Reply:
this is a temp solution when capc repo is not ready (in the middle of transfer). 
we need to change from github.com to kubernetes.sigs before release, I could not create a branch in CAPC's offical repo, so this may still be needed during dev period.